app/templates/admin/base_site.html

```diff
@@ -4,13 +4,16 @@
 {% block extrahead %}
 {{ block.super }}
     <link rel="stylesheet" type="text/css" href="{% static "admin/css/index.css" %}"/>
+    <link rel="stylesheet" type="text/css" href="https://cdn.jsdelivr.net/npm/daterangepicker/daterangepicker.css" />
     <script type="text/javascript" src="{% static "admin/js/vendor/jquery/jquery.min.js" %}"></script>
     <script type="text/javascript" src="{% static "admin/js/vendor/jquery/jquery.init.js" %}"></script>
     <script type="text/javascript" src="{% static "admin/js/vendor/globals.js" %}"></script>
     <script type="text/javascript" src="{% static "admin/js/vendor/core.js" %}"></script>
     <script type="text/javascript" src="{% static "admin/js/vendor/xregexp/xregexp.min.js" %}"></script>
     <script type="text/javascript" src="{% static "admin/js/vendor/calendar.js" %}"></script>
     <script type="text/javascript" src="{% static "admin/js/admin/DateTimeShortcuts.js" %}"></script>
+    <script type="text/javascript" src="https://cdn.jsdelivr.net/momentjs/latest/moment.min.js"></script>
```

Comment:
Put scripts/css in one place only. If used in multiple places, put in base_site. If not, put directly on the related page.